packages/field-base/test/label-mixin.test.js

```diff
@@ -96,10 +102,11 @@ describe('label-mixin', () => {
     describe('basic', () => {
       beforeEach(async () => {
         element = fixtureSync(`
-          <label-mixin-element>
+          <${tag}>
             <label slot="label">Custom</label>
-          </label-mixin-element>
+          </${tag}>
         `);
+        await nextRender();
         label = element.querySelector('label');
         await nextFrame();
```

Comment:
Is this `nextFrame` necessary? I guess the above `nextRender()` should be enough.